src/include/daos/checksum.h

```diff
@@ -15,48 +15,259 @@
  *
  * GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
  * The Government's rights to use, modify, reproduce, release, perform, display,
- * or disclose this software are subject to the terms of the Apache License
+ * or disclose this software are subject to the terms of the Apache License as
  * provided in Contract No. B609815.
  * Any reproduction of computer software, computer software documentation, or
  * portions thereof marked with this legend must also reproduce the markings.
  */
 
-#ifndef __DAOS_CHECKSUM_H
-#define __DAOS_CHECKSUM_H
+#ifndef __DAOS_CHECKSUMMER_H
```

Comment:
just curious why change to Checksummer?

Reply:
Initially I had added a new file (checksummer.h) but then replaced checksum.h with it, renaming checksummer -> checksum, but missed the header guard. Thank you for catching it, I'll change it to match the file name.